util/design/gen-otp-mmap.py

```diff
@@ -85,6 +85,7 @@ def main():
 
         with open(MMAP_TABLE_FILE, 'w') as outfile:
             outfile.write(TABLE_HEADER_COMMENT + otp_mmap.create_mmap_table())
+            outfile.write("\n")
```

Comment:
@msfschaffner i had to add this since the `trailing_whitespace_check` was failing otherwise.